pycyphal/transport/can/media/socketcan/_socketcan.py

```diff
@@ -319,11 +321,32 @@ class _NativeFrameDataCapacity(enum.IntEnum):
 _CAN_EFF_MASK = 0x1FFFFFFF
 
 
-def _make_socket(iface_name: str, can_fd: bool) -> socket.socket:
+def _get_tx_queue_len(iface_name: str) -> int:
+    try:
+        sysfs_net = pathlib.Path("/sys/class/net/")
+        sysfs_tx_queue_len = sysfs_net / iface_name / "tx_queue_len"
+        return int(sysfs_tx_queue_len.read_text())
+    except FileNotFoundError as e:
+        raise FileNotFoundError("tx_queue_len sysfs location not found") from e
```

Comment:
What is the point of catching and raising the same exception here? Are you sure we need this?

Reply:
Okay, I see.